src/mlpack/bindings/R/mlpack/DESCRIPTION.in

```diff
@@ -0,0 +1,24 @@
+Package: mlpack
+Title: 'Rcpp' Integration for the 'MLPACK' Library
+Version: ${PACKAGE_VERSION}
+Date: ${PACKAGE_DATE}
+Author: mlpack Team
+Maintainer: mlpack Team <mlpack-git@lists.mlpack.org>
+Description: The mlpack is a fast, flexible machine learning library, written
```

Comment:
Need to wrap `mlpack` in quotations, e.g. `'mlpack'`. Also, likely need to do the same for `C++`, ... 

I would remove: 

```
mlpack provides these
             algorithms as simple command-line programs, C++ classes and bindings
             for : Python, Julia, Go and R which can then be integrated into
             larger-scale machine learning solutions.
```

See the descriptions for Rcpp/RcppArmadillo/RcppEnsmallen for examples of quotations. (This will come up on submission to CRAN by a manual reviewer.)

- https://cran.r-project.org/package=Rcpp
- https://cran.r-project.org/package=RcppArmadillo
- https://cran.r-project.org/package=RcppEnsmallen

Reply:
I noticed a few other warts in the package when I rebuilt it today.  We'll get there, don't need to cram it in here today.

Reply:
Done.